scripts/gatkcondaenv.yml.template

```diff
@@ -44,8 +44,8 @@ dependencies:
   - scipy==1.0.0
   - six==1.11.0
   - tensorflow==1.4.0
-  - tensorflow-tensorboard==0.4.0rc3
+  - $tensorFlowDependency
```

Comment:
Hm - this needs to be fixed - this line should replace the tensorflow dependency, not the the tensorboard dependency.

Reply:
Done.